projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -507,16 +506,12 @@ function get_logo() {
 		);
 
 		// Get current site logo values.
-		$logo = get_option( 'site_logo' );
+		$logo_id = get_option( 'site_logo' );
 
 		// Update the response array if there's a site logo currenty active.
-		if ( $logo && 0 != $logo['id'] ) {
-			$logo_setting['id']  = $logo['id'];
-			$logo_setting['url'] = $logo['url'];
-
-			foreach ( $logo['sizes'] as $size => $properties ) {
-				$logo_setting['sizes'][ $size ] = $properties;
-			}
+		if ( $logo_id ) {
```

Comment:
Do we need to update the `sizes` value here as well?

Reply:
We could, but it's hard for me to believe any clients are actually using this: it would be empty for any sites using a more recent theme that uses the `custom_logo` theme mod, rather than the legacy `site_logo` option.